OxfordBodleian/Juel-Jensen/BDLaethe30.xml

```diff
@@ -181,44 +590,177 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                    </history>
                     
                     <additional>
                         <adminInfo>
                             <recordHist>
                                 <source>
-                                  <!--  <listBibl type="secondary">
-                                        <bibl></bibl>
-                                    </listBibl>-->
+                                    <listBibl type="secondary">
+                                        <bibl>
+                                            <ptr target="bm:Bassano1922BiluyKidan"/>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:DegeMueller2015Psalter"/>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Devens2007Mahaleya"/>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Fries1892Weddase"/>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Getatchew2003Anqasa"/>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Grohmann1919Marien"/>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Gleave1951Songof"/>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Heldman2010Psalter"/>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Knibb2000Psalms"/>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Lash1972Gateof"/>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Ludolf1701Psalterium"/>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Tasfa1966Dawit"/>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Weninger2010EAEWeddaseMaryam"/>
+                                        </bibl>
+                                        <!-- MV 26.03.20: Should some of these bibl references go under <listBibl type="editions"/>? Or is the latter destined to records with text editions only? -->
```

Comment:
the comment should be removed from the file

Reply:
thank you both for comments. I think we'll have to discuss a bit how to deal with the bibliography, because - although rules on MSS records are now a bit stricter - printed catalogues customarily display bibliography on the works contained in the manuscripts (and I'd say more usually on them rather than on those very mss, since by definition catalogues generally deal with previously unknown manuscripts). I will give all bibliography under "secondary list" for now.